app/client/src/components/pages/Community/components/tabs/Restore.js

```diff
@@ -165,23 +150,14 @@ function Restore() {
             </StyledNumber>
           )}
           <StyledLabel>Plans</StyledLabel>
-          <div css={switchContainerStyles}>
-            <Switch
-              checked={Boolean(waterbodyCount) && restoreLayerEnabled}
-              onChange={(checked) => {
-                setRestoreLayerVisibility(!restoreLayerEnabled);
-              }}
-              disabled={!Boolean(waterbodyCount)}
-              ariaLabel="Waterbodies"
-            />
-          </div>
         </StyledMetric>
       </StyledMetrics>
 
       <ContentTabs>
         <Tabs
           onChange={(index) => {
             if (index === 1) setRestoreLayerVisibility(true);
+            else setRestoreLayerVisibility(false);
           }}
         >
```

Comment:
Since the if/else statement here is setting a boolean value based on if index equals 1, you can simplify this by passing that condition in as the argument to the function:

```suggestion
        <Tabs onChange={(index) => setRestoreLayerVisibility(index === 1)}>
```

Reply:
Done. Looks much cleaner.